src/utils/BatchManager.js

```diff
@@ -96,6 +96,7 @@ class BatchManager {
         duration: null,
         html: null,
         returnMeta: {},
+        resLocals: response.locals,
```

Comment:
specifically, should we just spread `baseContext` into here?

Reply:
Passing in `request` / `response` sounds good!

Could pass this is in explicitly - spreading in `baseContext` sounds good too, at the risk of potentially increasing the public api surface area.

I'll send another PR doing this.

(Side note - unclear what batchMeta is used for? Can this be removed?)
```
╔═ markl@dev21-uswest1cdevc: ~/oss/hypernova git:(add_response_object_to_context)
╚═ ♪ git --no-pager grep batchMeta
src/utils/BatchManager.js:75:      batchMeta: {},
```

Reply:
I don't think it's being used.

Reply:
>  should we just spread baseContext into here?

We could change [line 147](https://github.com/magicmark/hypernova/blob/2d96928e3817fd444f088c96f4629a5ed6aeff98/src/utils/BatchManager.js#L147) to be

```js
const context = this.getRequestContext(null, token);
```

It would be a breaking change but then the context passed in would have access to `baseContext` as well.

The downside is that the whole response object would be made available which means that consumers may be able to hijack the response.